test/type.coffee

```diff
@@ -8,19 +8,33 @@ Amen.describe "Type functions", (context) ->
     isString, isFunction, isObject, isArray, isRegExp, isDate, isDefined,
     isGenerator, isPromise} = require "../src/type"
 
-  context.test "type"
-
-  context.test "isType"
-
-  context.test "instanceOf"
+  context.test "type", ->
+    assert type 7
+    assert type "7"
+    assert type true
+
+  context.test "isType", ->
+    assert isType(String, "7")
+    assert isType(Number, 7)
+    assert isType(Boolean, true)
+    assert ! isType(Number, false)
+
+  context.test "instanceOf", ->
```

Comment:
This test should look a little different.  Dan just converted [JavaScript's instanceof operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof) from a binary operator into a function so it plays nice with functional programming.

JavaScript's native form of object inheritance uses prototypes, so you get this history chain of prototypes from an object's ancestors.  `instanceof` checks that chain and returns `true` if it finds what it's looking for or `false` if it does not.  If you look at that above link, the test should look a little more like example they give.  MDN is awesome :)